scripts/webpack/webpack.dev.js

```diff
@@ -89,7 +89,17 @@ module.exports = (env = {}) =>
         : new ForkTsCheckerWebpackPlugin({
             eslint: {
               enabled: true,
-              files: ['public/app/**/*.{ts,tsx}', 'e2e/suite1/**/*.{ts,tsx}', 'public/test/**/*.{ts,tsx}'],
+              files: [
+                'public/app/**/*.{ts,tsx}',
+                'packages/grafana-ui/src/**/*.{ts,tsx}',
+                'packages/grafana-data/src/**/*.{ts,tsx}',
+                'packages/grafana-runtime/src/**/*.{ts,tsx}',
+                'packages/grafana-e2e-selectors/src/**/*.{ts,tsx}',
+                'packages/jaeger-ui-components/src/**/*.{ts,tsx}',
```

Comment:
wouldn't `packages/**/*.{ts,tsx}` suffice?

Reply:
good catch but sadly I couldn’t make that work. It failed with an error I couldn’t manage to solve.

Reply:
Okay, maybe add a comment explaining why. Aside from that, the changes LGTM.